input/ACTCMB_BN.yml

```diff
@@ -0,0 +1,8 @@
+mapper_class: MapperACTCMB
+mask_name: mask_ACT_compsep_BN
+path_rerun: '/mnt/extraspace/damonge/Datasets/ACT_DR4/xcell_runs'
+coords: G
+file_map: /mnt/extraspace/damonge/Datasets/ACT_DR4/compsep_maps/tilec_single_tile_BN_cmb_map_v1.2.0_joint.fits
+file_mask: /mnt/extraspace/damonge/Datasets/ACT_DR4/masks/compsep_masks/act_dr4.01_s14s15_BN_compsep_mask.fits
+map_name: ACT_CMB_BN
+lmax: 6000
```

Comment:
Should there be a `beam_info` section for this mapper?

Reply:
Yes and in all ACT CMB and tSZ